sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

```diff
@@ -57,6 +58,41 @@ The UUID can be obtained using the standard get\_info() interface.
 
 
 
+# PCI Device ID #
+
+A new device descriptor will be added which will provide the 16-bit PCI device ID.
```

Comment:
The return type of the information descriptor is defined as `uint32_t`: do we plan to require this API to return a value in a single format for all backends or each backend is free to return the value in its specific format?

In any case, I think the wording here should be adjusted a bit either to say which exact 16 bits within a 32 bit value correspond to PCI device ID or that the returned value is backend-specific (and therefore it should probably avoid mentioning 16 bits at all).

Reply:
> Absolutely! That seems like a reasonable requirement that the OpenCL extension seems to agree with. The problem here boils down to the fact that this extension is currently promising more than the used OpenCL query does, without doing any reliable checks or filtering for values that do not adhere to that promise (i.e. values that are not PCI device IDs).

Then the solution is straightforward. Rename the aspect to `ext_intel_device_id` and copy the specs from the OpenCL extension, which is the following:

> A unique device identifier for the OpenCL device.
> 
> If the implementation is driven primarily by a PCI device with a PCI device ID, the low 16 bits of the device identifier must contain that PCI device ID, and the remaining bits must be set to zero. Otherwise, the choice of what to return may be dictated by operating system or platform policies - but should uniquely identify both the device version and any major configuration options (for example, core count in the case of multi-core devices).

Thus the implementation of `ext_intel_device_id` would be perfect for both L0 and OpenCL. Do you have any objections?